index.html

```diff
@@ -26,5 +27,121 @@ <h2 class="intro-section-item" id="intro-text2">I’am a Software Developer</h2>
+          <li class="card-lang-item card-lang-bootstrap">bootstrap</li>
+          <li class="card-lang-item card-lang-ruby">Ruby</li>
+        </ul>
+        <button type="button" class="card-project-item card-project-button buttons">See Project</button>
+      </div>
+      <div class="card-project-container card-grid-item4">
+        <h1 class="card-project-item card-project-title">Professional Art Printing Data</h1>
+        <p class="card-project-item card-project-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card-project-item card-lang-container">
+          <li class="card-lang-item card-lang-html">html</li>
+          <li class="card-lang-item card-lang-bootstrap">bootstrap</li>
+          <li class="card-lang-item card-lang-ruby">Ruby</li>
+        </ul>
+        <button type="button" class="card-project-item card-project-button buttons">See Project</button>
+      </div>
+      <div class="card-project-container card-grid-item5">
+        <h1 class="card-project-item card-project-title">Professional Art Printing Data</h1>
+        <p class="card-project-item card-project-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card-project-item card-lang-container">
+          <li class="card-lang-item card-lang-html">html</li>
+          <li class="card-lang-item card-lang-bootstrap">bootstrap</li>
+          <li class="card-lang-item card-lang-ruby">Ruby</li>
+        </ul>
+        <button type="button" class="card-project-item card-project-button buttons">See Project</button>
+      </div>
+      <div class="card-project-container card-grid-item6">
+        <h1 class="card-project-item card-project-title">Professional Art Printing Data</h1>
+        <p class="card-project-item card-project-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="card-project-item card-lang-container">
+          <li class="card-lang-item card-lang-html">html</li>
+          <li class="card-lang-item card-lang-bootstrap">bootstrap</li>
+          <li class="card-lang-item card-lang-ruby">Ruby</li>
+        </ul>
+        <button type="button" class="card-project-item card-project-button buttons">See Project</button>
+      </div>
+    </section>
+  </section>
+
+  <section class="aboutme">
+    <div class="firsthalf-container">
+      <h1 class="firsthalf-item firsthalf-title">About me</h1>
+      <p class="firsthalf-item firsthalf-para">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button type="button" class="firsthalf-conatiner firsthalf-button buttons">Get My Resume</button>
```

Comment:
Please activate the button styling: Active, Hover and Pressed on this button. Its a part of the project requirement. Follow he guide given by the figma design
<img width="813" alt="Screenshot 2022-10-28 at 23 29 49" src="https://user-images.githubusercontent.com/48546992/198726809-51489ba0-6fc6-40bd-909b-a2814944d88b.png">

Reply:
heya, @Nyame-Wolf Hover, pressed and disabled are already added on the button. I am recording the screen and will send it here and in slack.

Reply:
https://user-images.githubusercontent.com/43726580/198731894-860d2031-a65b-4148-8b44-270f9540ba05.mp4

Reply:
Hi @Anand8317 
Your project is fine. You can make changes on the headings and ask for a review. Sorry for the oversight.